README.md

```diff
@@ -95,6 +95,7 @@ Run `make help` to see all the possible targets and variables:
     TESSDATA_REPO      Tesseract model repo to use. Default: _fast
     GROUND_TRUTH_DIR   Ground truth directory. Default: data/ground-truth
     MAX_ITERATIONS     Max iterations. Default: 10000
+    NET_SPEC           Network specification. Default: [1,36,0,1 Ct3,3,16 Mp3,3 Lfys48 Lfx96 Lrx96 Lfx256 O1c1]
```

Comment:
Not exactly true, `c1` is a variable.

Reply:
Would using a well defined placeholder (for example `c###` instead of `c1`) be better? That would also solve your question above.

I had chosen `c1` because that's how Tesseract currently documents the network specifications which were used in the models.

Reply:
I do not know the net-spec language well enough to judge whether `c1` is really a risk... `c###` is definitely safe.

Reply:
I'll add a commit that does this.